layout/mainmenu.xml

```diff
@@ -128,76 +118,102 @@
 			</Panel>
 		</Panel>
 		
-		<Panel id="MainMenuTopBarWorkshopContent" class="MainMenuTopBarContent" hittest="true">
+		<Panel id="MainMenuButtonsWorkshopContent" class="MainMenuButtonsContent" hittest="true">
 			<!-- todo: addons panel -->
 			<Label style="align: center center;" text="Steam workshop functionality is not currently implemented." />
 		</Panel>
-		<Panel id="MainMenuTopBarSettingsContent" class="MainMenuTopBarContent" hittest="true">
+		
+		<Panel id="MainMenuButtonsNewGameContent" class="MainMenuButtonsContent hide">
+			<Label class="MainMenuButtonsNewGameLabel" text="Start New Game" />
+			<Panel class="full flow-right">
+				<Panel class="MainNewGameList" id="MainMenuButtonsNewGameList">
+				</Panel>
+				<Panel class="full MainNewGamePanel">
+					<Panel id="MainNewGamePanel" class="full hide">
+						<Image id="MainMenuNewGameCampaignImage"/>
+						<Label text=" Chapters:"/>
+						<Label class="full" id="MainMenuNewGameChapterTitleLabel"/>
+						<Label class="full" id="MainMenuNewGameChapterTitle2Label"/>
+						<Label class="full" id="MainMenuNewGameChapterDescriptionLabel"/>
+						<Panel class="full flow-right">
+							<Panel class="MainMenuNewGameContainer flow-down" id="MainMenuButtonsNewGamePanel">
+							</Panel>
+							<Image id="MainMenuNewGameChapterImage" scaling="stretch"/>
+						</Panel>
+						<Button disabled="true" onactivate="MainMenuController.playSelectedMap()" hittest="true" id="MainMenuNewGamePlayButton" class="ListItem MainMenuNewGamePlayChapterButton">
+							<Label text="Start Game"/>
+						</Button>
```

Comment:
nit: this is why `TextButton` exists

Reply:
yes it is, and they function ever so differently, SASS didn't want to listen to me for whatever reason, so I opted for this instead.